src/styles/_catalog.scss

```diff
@@ -1,22 +1,27 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+@mixin responsive($minWidth, $howRepeat) {
+  @media (min-width: #{$minWidth}) {
+    grid-template-columns: repeat($howRepeat, minmax(200px, 1fr));
```

Comment:
```suggestion
    grid-template-columns: repeat($howRepeat, 200px);
```
bc now it looks wrong, move 200px into variable
<img width="741" alt="image" src="https://user-images.githubusercontent.com/53611910/206903646-6ffa96f9-ee55-4da3-a42d-c1f837d032e9.png">